blog/_posts/2017-11-30-bloop-release.md

```diff
@@ -0,0 +1,153 @@
+and `console`), and allows other build tools to integrate with it.
+
+In practice, this means that when users are only going to compile and test their project, they can
+use `bloop` instead of a full-blown resource-hungry build tool; and that generic build tools can
+delegate to `bloop` to compile and test users' projects instead of creating their own Scala and Java
+integrations.
+
+## The current status
+
+As of this announcement, `bloop` is capable of compiling and testing your project, and it integrates
+with `sbt`. It supports all the test frameworks sbt supports (`JUnit`, `Scalatest`, `Scalacheck`,
+`utest` and more).
+
+`bloop` is a young project with only three weeks of life. That means that there's some work left
+before we support all different platforms and needs. In our next release, we want to focus on:
+
+- Rethinking from scratch the internal APIs and stabilize the project.
+- Supporting `console`, [a faster `testQuick`](https://github.com/scalacenter/bloop/issues/61), and
+  [built-in file watching](https://github.com/scalacenter/bloop/issues/7).
+- Making more performance improvements to our integration, to be even faster.
+- Supporting Dotty and out-of-the-box cross-compilation to
+  [Scala.js](https://github.com/scala-js/scala-js/) and [Scala
+  Native](https://github.com/scala-native/scala-native).
+* Exploring integrations with other build tools like `Maven` and `Gradle`.
+
+### First beta release `0.1`
+
+The first beta version of bloop is out in Maven Central. To add it to your project, head over
+[scalacenter/bloop][].
+
+### Preserving hot compilers
+
+An important feature of `bloop` is its [nailgun][] integration. Nailgun is a client, protocol, and
+server for running Java programs from the command line without incurring the JVM startup overhead.
+With it, `bloop` ensures that you always have a hot compiler in the background to compile your
+project. This brings significant improvements indeed: [a hot compiler can compile up to 18 times
+faster than a hot
+compiler](https://scala-ci.typesafe.com/grafana/dashboard/db/scala-benchmark?var-branch=2.12.x&var-source=vector&var-bench=HotScalacBenchmark.compile&var-host=scalabench@scalabench@).
+
+Keeping a hot compiler in the background is crucial if one takes compilation speed and productivity
+seriously. This is often difficult and takes space of mind. For example, `sbt` throws away a hot
+compiler every time users quit (`Ctrl-C`) and restart, but also when they do `reload` after several
+executions of `compile`. By this rule, the more users modify their sbt build, the less productive
+they are.
+
+Current sbt users can indeed avoid quitting and prefer suspending (`Ctrl-Z`), but they will always
+be punished for modifying their builds. In our experience, changing habits is hard, so we have
+preferred to add a built-in nailgul integration in `bloop` that addresses this problem at its root
+rather than changing people's habits, which is hard and time consuming.
```

Comment:
It's still not clear though what you did to a completely novice reader. You added nailgun to `bloop` instead of sbt because why? Why is using nailgun faster than just using sbt and suspending with `Ctrl-Z`?

Reply:
I've removed this part altogether, it was too confusing